packages/grafana-ui/src/utils/handleReduceMotion.ts

```diff
@@ -0,0 +1,11 @@
+import { CSSInterpolation } from '@emotion/css';
+
+export const handleReduceMotion = (styles: CSSInterpolation, reduceMotionStyles?: CSSInterpolation) => {
```

Comment:
minor nit, think we should use `reducedMotion` always to match the media query name
```suggestion
export const handleReducedMotion = (styles: CSSInterpolation, reducedMotionStyles?: CSSInterpolation) => {
```

Reply:
I think I always write it wrong 😅